Source/Extensions/StringExtensions.swift

```diff
@@ -13,6 +13,10 @@ extension String {
     func standardize() -> String {
         return self.replacingOccurrences(of: "[^0-9]", with: "", options: .regularExpression)
     }
+
+    func decimalDigits() -> String {
+        components(separatedBy: .decimalDigits.inverted).joined()
+    }
```

Comment:
I am unsure what this is doing. Some unit tests would also be nice to clarify why it is needed.

Reply:
It's returning only digits without special characters. It's Native Apple API , no need for unit test

Reply:
`decimalDigits` is not Apple API though, its our own method that uses 2 Apple APIs together to create our own transformation.

Also why the name _decimal digits_ and not just _digits_?